src/app/modules/about-module/layout.tsx

```diff
@@ -165,9 +165,9 @@ export const AboutLayout = () => {
             </Typography>
             <ParagraphContainer data-cy="p6">
               <Typography variant="body1">
-                The benefit of the IATI Datastore is that you can convert the
-                original IATI data, provided in XML format, into two more
-                formats - CSV and JSON. All data in the Datastore is filtered
+                The benefit of IATI.cloud is that you can convert the
+                original IATI data, provided in XML format, into three 
+                formats - CSV, XLM and JSON. All data in IATI.cloud is filtered
```

Comment:
XML

Reply:
Keep it for now and push, will review on a next iteration now I have this running locally..